src/components/visualizations/chart/highcharts/helpers.ts

```diff
@@ -137,3 +155,71 @@ export function getDataLabelAttributes(point: any) {
         height: 0
     };
 }
+
+export function shouldFollowPointer(chartOptions: any) {
+    // Sets true if max or min is set on any axis from properties
+    // and is lower than max data value from series
+    const type = chartOptions.type;
+
+    if (!isColumnChart(type) && !isBarChart(type)) {
+        return false;
+    }
+
+    const xMax = Number(get(chartOptions, 'xAxisProps.max'));
+    const yMax = Number(get(chartOptions, 'yAxisProps.max'));
+    const xMin = Number(get(chartOptions, 'xAxisProps.min'));
+    const yMin = Number(get(chartOptions, 'yAxisProps.min'));
+
+    // Return false when no extrem is set in properties
+    if (!xMax && !yMax && !xMin && !yMin) {
+        return false;
+    }
+
+    if (isBarChart(type) && !xMax && !xMin) {
+        return false;
+    }
+
+    if (!isBarChart(type) && !yMax && !yMin) {
+        return false;
+    }
+
+    const maxDataValue = getExtremeDataValue(chartOptions, true);
+    const minDataValue = getExtremeDataValue(chartOptions, false);
+
+    if (isBarChart(type)) {
+        return (xMax && maxDataValue > xMax) || (xMin && minDataValue < xMin);
+    }
+
+    return (yMax && maxDataValue > yMax) || (yMin && minDataValue < yMin);
+}
+
+function getNonStackedExtremeValue(chartOptions: any, isMax: boolean): number {
+    return chartOptions.data.series.reduce((acc: number, serie: ISeriesItem) => {
+        const extremeSerieValue = getSerieDataExtremeValue(serie.data, isMax).y;
+        return acc > extremeSerieValue ? acc : extremeSerieValue;
```

Comment:
shouldn't you take into account `isMax` variable when comparing values? ie. `isMax ?` use `>`, else use `<`

Reply:
Yes I looked on this and you are probably right. But why this is woks then? :D I will investigate.